common/views/components/Tasl/Tasl.js

```diff
@@ -167,7 +168,10 @@ const Tasl = withToggler(
               properties: ['padding-top', 'padding-bottom'],
             }}
             h={{ size: 's', properties: ['padding-left'] }}
-            className={`drawer__body bg-black font-white`}
```

Comment:
`drawer__body` was previously responsible for this, but we removed the drawer scss. Rather than re-adding that file, this felt like the more appropriate way to handle it.